Jenkinsfile

```diff
@@ -0,0 +1,78 @@
+pipeline
+{
+    agent
+    {
+        node
+        {
+            label 'embedded-docker'
+        }
+    }
+    environment
+    {
+        MAKEFLAGS = '-j 3'
```

Comment:
makeflags should be autodetected by the build itself. what if we only get one core? having to many build jobs actually makes things worse. Or even worse, what if we allow 10 cores for the job to be done quicker, we'd be missing out.
thus we should let the build figure this out itself, which in effect thus also works locally.